charts/atlantis/templates/statefulset.yaml

```diff
@@ -42,6 +42,9 @@ spec:
       {{- end }}
       {{- end }}
       serviceAccountName: {{ template "atlantis.serviceAccountName" . }}
+      {{- if .Values.shareProcessNamespace }}
+      shareProcessNamespace: {{ .Values.shareProcessNamespace }}
+      {{- end }}
```

Comment:
If this defaults to `false` do we even need the if statement?

Reply:
Good point @nitrocode
Would you prefer having the if statement or removing the default from false? 

I think removing the if statement would be my preferred solution, but I'll do either.

Reply:
I think removing the if statement would be fine for now since it already defaults to `false`

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/

> Share a single process namespace between all of the containers in a pod. When this is set containers will be able to view and signal processes from other containers in the same pod, and the first process in each container will not be assigned PID 1. HostPID and ShareProcessNamespace cannot both be set. Optional: Default to false.

Reply:
It is now removed 👨🏻‍💻